### PR TITLE
Makes admin panel icon visibility conditional based on permissions

### PR DESCRIPTION
**Pull request by drzackyll**

### CODE REVIEW

File: etools-app-selector.html [hunk 1]
```diff
@@ -391,6 +398,13 @@
       }
 
       _renderIcons() {
+        let admin = false;
+        if (this.user.is_superuser === 'True') {
```
Comment (adi130987) [inline etools-app-selector.html]: make sure `this.user` is not `null`/`undefined` (see `_checkPermission` method)

Comment (adi130987) [reply etools-app-selector.html]: use just one condition (both are setting `admin = true`) or create a new method to check if the user is admin.

```
isAdminUser(user) {
    return user.is_superuser === 'True' || this.user.groups.find(group => ...)
}
```

Comment (adi130987) [reply etools-app-selector.html]: using `this.user.groups.find(group => group.id === '87')` is confusing and dvelopers will not understand what 87 represents (just an id). Use the name instead `Country Office Administrator` (maybe define it as a static readonly prop inside app selector class)
```
static readonly COUNTRY_OFFICE_ADMIN_GROUP_NAME = 'Country Office Administrator';

...

this.user.groups.find(group => group.name === EtoolsAppSelector.COUNTRY_OFFICE_ADMIN_GROUP_NAME)
```

**Comment by drzackyll** [comment]
@adi130987 hopefully this is resolved! let me know if there should be further changes.

**Comment by adi130987** [comment]
@drzackyll all 3 PRs are approved... please create new versions and update etools apps.

**Pull request merged**